helm/presets/aws.yaml

```diff
@@ -0,0 +1,44 @@
+receive:
+  service:
+    type: LoadBalancer
+broadcast:
+  service:
+    type: LoadBalancer
+acquire:
+  service:
+    type: LoadBalancer
+strimzi:
+  kafka:
+    resources:
+      requests:
+        cpu: 500m
+        memory: 6Gi
```

Comment:
you sure we have the space to turn this up to 6gigs?  Do we really need 6 gigs or would something like 4 be enough for a base install?

Reply:
I've deployed it and it was working for me.